src/AuthModule/Components/Login/Login.tsx

```diff
@@ -1,9 +1,8 @@
 import { useForm } from "react-hook-form";
 import logo from "../../../assets/PMS 3.svg";
-import React, { useContext, useState } from "react";
+import  { useContext, useState } from "react";
 import { Link, useNavigate } from "react-router-dom";
 import axios from "axios";
-// import { toast } from "react-toastify";
 import Loader from "../../../SharedMoudule/Components/Loading/Loading";
 import { AuthContext } from "../../../Context/Components/AuthContext";
 import { ToastContext } from "../../../Context/Components/ToastContext";
```

Comment:
I noticed having a ToastContext for handling `toastSuccess` , `toastError` , `toastAny`, however the `react-toastify` package provide us with `toast.success` and `toast.error` functions that we can use for this purpose. We only need the `ToastContainer` to a top-level component such as the `App.tsx` and then use these methods in any child component without the need to create a contet for this purpose.

and then we will nee to import the toast function only from the package:
![image](https://github.com/SalmaAboArab/Project-Management-System/assets/64327685/74999038-af80-4f2b-bb6f-7322d760ea05)